dborutils/batch_manager.py

```diff
@@ -1,130 +0,0 @@
-from datetime import datetime
-from decimal import Decimal
-import logging
-logger = logging.getLogger(__name__)
-
-
-class BatchManager(object):
```

Comment:
It was dropped in https://github.com/NoodleEducation/dborutils/pull/14 on top of which I built this and https://github.com/NoodleEducation/dborutils/pull/33. I was kinda expecting you to review them separately, since they might be more coherent that way? I'm not averse to merging this one only which will do everything at once.